slide-deck.js

```diff
@@ -503,7 +512,7 @@ class slideDeck extends HTMLElement {
   };
 
   goToSaved = () => {
-    this.goTo(this.slideFromStore());
+    this.goTo(this.slideFromStore() || 1);
```

Comment:
Currently `slideFromStore` is returning `NaN` or a number- I'm wondering if it would make sense to to move the `|| 1` into the `slideFromStore` function, so that if we need to get the slideFromStore in the future, we don't need to handle the `|| 1` elsewhere?

Reply:
Yeah, I wasn't sure which behavior we would want in the abstract? Do we want to know that the store is empty, and then we can decide how to fall back? Or do we just want a slide number, so the fallback is included? I suppose we could support both with something like:

```js
slideFromStore = (fallback = 1) => { … }
this.slideFromStore(fallbackOverrideValueIfNeeded)
```